src/cmd/cmd.js

```diff
@@ -156,7 +157,8 @@ class Cmd {
           logFile: options.logfile,
           logLevel: options.loglevel,
           webpackConfigName: options.pipeline || 'development',
-          openBrowser: !options.nobrowser ? null : false
+          openBrowser: !options.nobrowser ? null : false,
+          singleUseAuthToken: options.singleUseAuthToken
```

Comment:
I'm confused, where does `singleUseAuthToken` come from if the option is named `--no-single-use-auth-token`?